.github/workflows/contribution-check.yml

```diff
@@ -0,0 +1,16 @@
+name: Check PR preview
```

Comment:
**suggestion:** can we rename it into `Contribution checks` for consistency with the bonita-doc repository

Reply:
done in 11123a6c869422e64a99bf9fc5f127f36a355fd6